File: tests/openmp_examples/devices/target_update.2.c
#pragma omp target data map(to: v1[:N], v2[:N]) map(from: p[0:N])
#pragma omp target
#pragma omp parallel for
#pragma omp target update if (changed) to(v1[:N])
#pragma omp target update if (changed) to(v2[:N])
#pragma omp target
#pragma omp parallel for

File: tests/openmp_examples/devices/target_update.2.f90
!$omp    target data map(to: v1, v2) map(from: p)
!$omp       target
!$omp          parallel do
!$omp       end target
!$omp       target update if(changed) to(v1(:N))
!$omp       target update if(changed) to(v2(:N))
!$omp       target
!$omp          parallel do
!$omp       end target
!$omp    end target data
